springerlink.md

```diff
@@ -1060,9 +1020,9 @@ Third party advertising scripts may inject non-responsive content that loses or
 </ul></td>
 <td>Partially Supports</td>
 <td>
-Content in journals, books, series, protocols, reference works or proceedings may include complex images that contain images of text. In most cases these images are used to supplement the text content they appear alongside. In some cases (e.g. charts, graphs, or maps, among others), the text content presented alongside may be insufficient to explain the contents of the image.
+Outside of editorial content managed by content management system (CMS), the contents of author-submitted published books and papers, and third party scripts, the pages of this product do not use images of text to convey information except in the case of <a href="https://www.w3.org/WAI/WCAG21/Understanding/images-of-text.html">logotypes</a> (considered essential per WCAG 2.1) or when the image is <a href="https://www.w3.org/TR/WCAG21/#dfn-pure-decoration">pure decoration</a>.
```

Comment:
Might "**a** content management system" or "content management system**s**" be meant?
Also, I had trouble identifying where the enumeration ends and the sentence continues, so eventually a list could be useful here.

```suggestion
Outside of 
<ul>
  <li>editorial content managed by content management systems (CMS),</li>
  <li>the contents of author-submitted published books and papers, and</li>
  <li>third party scripts,</li>
</ul>
the pages of this product do not use images of text to convey information except in the case of <a href="https://www.w3.org/WAI/WCAG21/Understanding/images-of-text.html">logotypes</a> (considered essential per WCAG 2.1) or when the image is <a href="https://www.w3.org/TR/WCAG21/#dfn-pure-decoration">pure decoration</a>.

```

Reply:
Yes, an excellent suggestion. We only have the one CMS for this kind of thing so I'll push up an amendment separately, thank you!